### PR TITLE
Add an article and a video

### PR DESCRIPTION
**Pull request by manishsaraan**
I have added new links for AVA articles and videos. I will search for more links later.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -21,13 +21,13 @@
 - [Recipes](https://github.com/avajs/ava/tree/master/docs/recipes)
 - [Testing React Native apps with AVA](https://shift.infinite.red/testing-the-bejeezus-out-of-react-native-apps-with-ava-330f51f8f6c3)
 - [Getting Started with Create React App and AVA](https://semaphoreci.com/community/tutorials/getting-started-with-create-react-app-and-ava)
-
+- [Effortless unit testing with AVA](https://wecodetheweb.com/2016/04/19/effortless-unit-testing-with-ava/)
```
Comment (kevva) [inline readme.md]: Could you keep the (two) blank lines between each section?

**Review by kevva — CHANGES_REQUESTED** [review]

**Comment by manishsaraan** [comment]
I have made the changes. Can you review it again. Thanks.

File: readme.md [hunk 2]
```diff
@@ -21,12 +21,14 @@
...
 - [Testing React Native apps with AVA](https://shift.infinite.red/testing-the-bejeezus-out-of-react-native-apps-with-ava-330f51f8f6c3)
 - [Getting Started with Create React App and AVA](https://semaphoreci.com/community/tutorials/getting-started-with-create-react-app-and-ava)
+- [Effortless unit testing with AVA](https://wecodetheweb.com/2016/04/19/effortless-unit-testing-with-ava/)
 
 
 ## Videos
 
 - [JavaScript Air episode with the AVA team](http://jsair.io/ava)
 - [AVA Casts](http://avacasts.com) - Short screencasts about AVA.
+- [AVA Tutorial](https://www.youtube.com/watch?v=RxLW6-3dk5A) - AVA Tutorial Youtube Link.
```
Comment (sindresorhus) [inline readme.md]: This title too generic. The video look to be more about testing React components with AVA.

Comment (sindresorhus) [reply readme.md]: The description also needs to be improved. And `Youtube` => `YouTube`.

Comment (manishsaraan) [reply readme.md]: Thanks @sindresorhus . So you want me to update the description of youtube link. This is my first pull request, so I am bit of struggling here.

Comment (sindresorhus) [reply readme.md]: Specifically, `AVA Tutorial` is too generic, and `AVA Tutorial Youtube Link.` is not a good description.

Comment (manishsaraan) [reply readme.md]: Thanks for reply @sindresorhus . I will make the changes in pr.

**Comment by manishsaraan** [comment]
I have updated the description for video link.

**Review by kevva — APPROVED** [review]

**Comment by manishsaraan** [comment]
@sindresorhus . Can you merge the changes.

**Comment by manishsaraan** [comment]
@sindresorhus  can you merge the changes.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**